Packs/FancyEmails/Scripts/SendFancyEmailIncidentStatus/SendFancyEmailIncidentStatus.py

```diff
@@ -0,0 +1,95 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+args = demisto.args()
+incident = demisto.incident()
+incident_headers = argToList(args.get('incident_headers'))
+incident_fields = argToList(args.get('incident_fields'))
+include_notes = argToBoolean(args.get('include_notes'))
+banner = args.get('banner')
+indicator_query = args.get('indicator_query', False)
+custom_css = args.get('custom_css', '')
+indicator_table_params = {
+    'query': indicator_query,
+    'name': args.get('indicator_table_name'),
+    'use_alt_link': args.get('use_alt_link'),
+    'max_name_chars': args.get('max_name_chars')
+}
+
```

Comment:
Please use a main function, that will be getting the arguments and running a different function handling all the logic.
When missing, It might cause issues in the future when someone tries to import the code lines to another file.
Also, please add some unitest if possible.